style.css

```diff
@@ -1,414 +1 @@
-html,
```

Comment:
Why did you remove all styles from this file?
And also why didn't you just delete the entire file instead of leaving it empty?